src/Microsoft.AspNetCore.HttpOverrides/ForwardedHeadersOptions.cs

```diff
@@ -10,6 +10,38 @@ namespace Microsoft.AspNetCore.Builder
     public class ForwardedHeadersOptions
     {
         /// <summary>
+        /// X-Forwarded-For
+        /// </summary>
+        public const string DefaultForwardedForHeaderName = "X-Forwarded-For";
```

Comment:
We have precedence for separating these to their own class.
https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationDefaults.cs
How about ForwardedHeaderNames.XForwardedFor, etc..